samples/sn/src/styles.scss

```diff
@@ -0,0 +1,91 @@
+// overwrite values should be written here
+
+$textColor: blue;
+$navbarBackgroundColor: pink;
+
+@mixin dv-button {
+  background: #1FB8CC;
+  color: $textColor;
+  padding: 0 6px 0 6px;
```

Comment:
do you need to provide these values if they are the same as the default ones?

Reply:
No, and currently we don't need this mixin at all - it was written as a sample. I removed it so it's clearer what we want to achieve here.